<launch>
    <param name="am_noise"      type="double" value="1e-2" />
    <param name="wm_noise"      type="double" value="1e-4" />
    <param name="ab_noise"  	type="double" value="1e-6" />
    <param name="wb_noise" 		type="double" value="1e-6" />

    <param name="I_p_Gps_x"       type="double" value="0.0" />
    <param name="I_p_Gps_y"       type="double" value="0.0" />
    <param name="I_p_Gps_z"       type="double" value="0.0" />

    <node name="nmea_topic_driver" pkg="nmea_navsat_driver" type="nmea_topic_driver" output="screen" />
    <node name="eskf_localization_node" pkg="eskf_localization" type="eskf_localization_node" output="screen" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find eskf_localization)/ros_wrapper/rviz/rviz_config.rviz" output="screen"/>

</launch>